### ISSUE TITLE
Stats issue for changetsets that is created at an hour and closed in the next hour

### ISSUE THREAD

**Issue by emi420** [hotosm/insights#22]
Ex, for changeset: https://www.openstreetmap.org/api/0.6/changeset/78101045/download

The Insights DB has the following stats:

```
changeset | added_buildings | modified_buildings
----------+-----------------+-------------------
 78101045 |              24 |                0 |
```

But we're getting different results with another tool (Underpass):

```
{"changeset_id":78101045, "added":[{"building":24}], "modified": []},
{"changeset_id":78101045, "added": [], "modified":[{"building":27}]}
```

The `modified_buildings` in the Insights DB looks wrong, because in the changeset file there are 51 created/modified buildings (24+27) and the query that's doing the calculation for Insights also has the same result:

![image](https://user-images.githubusercontent.com/1226194/164459583-c85eb95c-4e81-4b3f-8b21-3bbde5a34ab6.png)

cc @omranlm

**Comment by omranlm** [comment]
Since the bug is in the create function, the affected changesets are only those before replication load in insights, so before 5 Sept 2021 in the maintain current env. The create function has been fixed in commit https://github.com/hotosm/insights/commit/2fe1fec1efa1c9f8666376415218c2cd6d404c10

**Comment by omranlm** [comment]
In the insights DB, after deployment of latest fix, running the command `python3 changesetStats.py -c 110750000` it will scan all change sets from changeset [#110750000 ](https://www.openstreetmap.org/changeset/110750000) on 5th Sept 2021 backward till changeset #1 
It is required to run the hashtag calculation (`python3 hashtags.py`) again after the first command & emptying `hashtag_stats` table to guarantee better accuracy if hashtag statistics

**Comment by omranlm** [comment]
Additionally, multiple threads can be initiated to scan/fix changesets 110750000 backward by running the same script multiple time 
`python3 changesetStats.py -c 110750000`
`python3 changesetStats.py -c 100000000`

In such scenario, manual stop for the first script would be needed when it finishes scanning changesets from 110750000 to 100000000

**Comment by omranlm** [comment]
Implementation wise, it takes ~2hours to scan/fix 50,000 changesets and some time less depending on the size of addedd,modified,deleted osm elements in the changesets 

![image](https://user-images.githubusercontent.com/39491329/164703970-7eae86f9-7d2e-4a5d-b83c-7f8de57c4527.png)
